src/main/java/uk/gov/pay/directdebit/notifications/services/UserNotificationService.java

```diff
@@ -42,7 +42,7 @@ public UserNotificationService(AdminUsersClient adminUsersClient, DirectDebitCon
     public void sendMandateFailedEmailFor(Mandate mandate) {
         adminUsersClient.sendEmail(EmailTemplate.MANDATE_FAILED, mandate,
                 ImmutableMap.of(
-                        MANDATE_REFERENCE_KEY, mandate.getMandateBankStatementReference().toString(),
+                        MANDATE_REFERENCE_KEY, mandate.getMandateBankStatementReference().map(x -> x.toString()).orElse(""),
```

Comment:
Will this show up in the email in an acceptable way? Or would it be better to give some other string instead which would be clearer in the email?

Reply:
I don't have the answer to this question but given it was possible the mandate bank statement ref was populated by a randomly generated value by dd-connector I don't think this is any worse than it was.

Reply:
Would be good to check the email template I reckon, it'd be quicker to fix now if necessary than it coming back to bite us later